code/src/components/QuizResult.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import dogsurfer from './images/dogiesurf.png'
+import shark from './images/shark.png'
+import jelly from './images/jelly.png'
+import crab from './images/krabba.png'
+import goat from './images/goat.png'
+import nakedcat from './images/nakedcat.png'
+import nakeddog from './images/nakenhund.png'
+import dogshark from './images/sharkdoggie.png'
+import dove from './images/Duva.png'
+
+export const QuizResult = ({ prefer, color, thing, name, setResults, results }) => {
+  const quiz = () => {
+    if (prefer === 'surf') {
+      if (color === 'black') {
+        if (thing === 'ocean') {
+          return (
+            <div className="Results">
+              Dum dum dum dum Shark <img src={shark} alt="Shark" />
+            </div>
+          );
+        } else if (thing === 'city') {
+          return (
+            <div className="Results">
+              Its a dog surfing <img src={dogsurfer} alt="Dog surfing" />
+            </div>
+          );
+        } else if (thing === 'mountain') {
```

Comment:
I definitely understand that it was difficult having to figure out how to handle so many conditionals, so awesome job getting that to work!! I would just look into how to use ternary operators to able shorten the code and if there would be another way to organise it.